### PR TITLE
frontend: sharing menu

### LINKED ISSUE #215: extend content "sharing" functionalities
At the moment, sharing functionalities are in the content sub menu "teilen"

Let's include:
- Google Classroom
- Facebook
- Twitter
- Mail

And may be, we should replace the sharing functionalities. E.g. like in Khan Academy (below the head line) : https://www.khanacademy.org/math/trigonometry/trig-with-general-triangles/solving-general-triangles/a/laws-of-sines-and-cosines-review

### PR DESCRIPTION
**Pull request by FridoSa**
closes #215

### CODE REVIEW

**Comment by inyono** [comment]
For the future: please create a draft pull request when your code is not ready for review / merge yet :)

File: packages/public/frontend/package.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "start": "next",
     "start-storybook": "start-storybook -p 9000 -c .storybook -s ./public"
   },
-  "dependencies": {},
+  "dependencies": {
+    "exenv": "^1.2.2"
```
Comment (Entkenntnis) [inline packages/public/frontend/package.json]: please move to devDevs

Comment (inyono) [reply packages/public/frontend/package.json]: And please change that to `^1.0.0` (except if there's a special reason that this has to be 1.2.2 or newer)

**Comment by inyono** [comment]
What's the state of this PR?

File: packages/public/frontend/package.json (hunk 1 above)
Comment (Entkenntnis) [reply packages/public/frontend/package.json]: done

**Comment by FridoSa** [comment]
mebis-Link points to facebook

**Comment by elbotho** [comment]
Mebis:
For the share menu we will build links using the ids and copy them or offer to copy them into the clipboard. Sorry no fancy stuff here :)
URLs should look like this:
`https:de.serlo.org/1553?ContentOnly`

**Pull request closed without merging**